builders/interoperability/xcm/xc20/overview.md

```diff
@@ -7,44 +7,73 @@ description: Learn about the types of cross-chain assets on Moonbeam, in particu
+While Substrate assets are inherently interoperable, interacting with them requires using the Substrate API, which offers little visibility into the EVM. This creates a barrier for EVM-focused developers. To address this gap and extend Polkadot/Kusama’s native interoperability to the EVM, Moonbeam introduced the concept of XC-20s.
 
-XC-20s are a unique asset class on Moonbeam. It combines the power of Substrate assets (native interoperability) but allows users and developers to interact with them through a familiar [ERC-20 interface](/builders/interoperability/xcm/xc20/interact/#the-erc20-interface){target=\_blank}. On the EVM side, XC-20s follow the standard [ERC-20 interface](/builders/interoperability/xcm/xc20/interact/#the-erc20-interface){target=\_blank}, so smart contracts and users can easily interact with them, and no knowledge of Substrate is required. This ultimately provides greater flexibility for developers when working with these types of assets and allows seamless integrations with EVM-based smart contracts such as DEXs and lending platforms, among others. Moreover, developers can integrate XC-20s with regular [Ethereum development frameworks](/builders/ethereum/dev-env/){target=\_blank}, or dApps, and create connected contract strategies with such assets. Moreover, with the introduction of [RT2301](https://github.com/moonbeam-foundation/moonbeam/tree/runtime-2301){target=\_blank}, all ERC-20s are XCM-ready, meaning they can also be referred to as XC-20s.
+Initially, Moonbeam offered XC-20s as a specialized asset type, requiring cross-chain configuration during asset creation to enable full cross-chain capabilities. Now, since [runtime 2301](https://github.com/moonbeam-foundation/moonbeam/tree/runtime-2301){target=\_blank}, any ERC-20 token on Moonbeam can be configured for cross-chain transfers at any time effectively allowing all ERC-20s to function as XC-20s. Said differently, it is possible to make any ERC-20 an XC-20 at any time, by following the steps in the [XC Registration section](/builders/interoperability/xcm/xc-registration/){target=\_blank}.  
+
+XC-20s are a unique asset class on Moonbeam that blend the native interoperability of Substrate assets with a standard [ERC-20 interface](/builders/interoperability/xcm/xc20/interact/#the-erc20-interface){target=\_blank}. This means developers and users can interact with them just like any other ERC-20 token, with no Substrate knowledge needed, allowing seamless integration with EVM-based smart contracts (such as DEXs, lending platforms, and more) and familiar Ethereum development frameworks.
```

Comment:
I think this definition of XC-20s should be before the backstory of them needing to be configured, but not anymore as of runtime 2301.